src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   
+    <title>Todo App</title>
+</head>
+<body>
+    <div class="todo-container">
+        <div class="todo-title">
+            <h2>Today's Todo</h2>
+            <hr>
+        </div>
+        <div class="todos">
+            <form>
+                <input type="text" name="task" id="task" placeholder="Add to your list">
+            </form>
+            <div class="todo-lists">
+                <ul id="todo-lists">
+
+                </ul>
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] It seems that the UI does not yet look like the one in the project requirements, since there are some details that are missing, see screenshot below 👇 
- Original Design
![image](https://user-images.githubusercontent.com/55465133/134435971-dd80acf0-c72e-4a56-963e-beb0538a495f.png)

- Your design
![image](https://user-images.githubusercontent.com/55465133/134436033-d83bed61-70ec-4238-b68e-56e7dbcda4b0.png)

As you can see you are missing some stuff, kindly let's fix this 😉